DuckDuckGo/SettingsState.swift

```diff
@@ -78,7 +78,7 @@ struct SettingsState {
     // Features
     var debugModeEnabled: Bool
     var voiceSearchEnabled: Bool
-    var speechRecognitionEnabled: Bool
+    var speechRecognitionAvailable: Bool // Returns if the device has speech recognition available
```

Comment:
I changed this back to `speechRecognitionAvailable` because this is used to decide if we can show the settings, it doesn't mean the feature is enabled